### PR TITLE
RLink fixes #2026

### LINKED ISSUE #2026: Null ref in RLink called by dlgDeleteRowsColumns when no data frame
When opening delete rows/columns without dataframe (very sensible, which is why this is probably not very urgent), the function in RLink: FillComboDataFrame was trying to run 
 cboDataFrames.SelectedIndex = cboDataFrames.Items.IndexOf(frmEditor.grdData.CurrentWorksheet.Name)
at line 122, but frmEditor.grdData.CurrentWorksheet is Nothing so Null ref and badam, crash !  

Solving proposal: Probably add a reality check for frmEditor.grdData.CurrentWorksheet in the RLink ? The objectif is to set the default selected index to the dataframe that's active, so I guess the step can be merely skipped if there is no dataframe. 

If I don't hear back and no one is working on RLink currently I m happy to implement that.

### PR DESCRIPTION
**Pull request by FrancoisJRenaud**
testing in progress

### CODE REVIEW

**Comment by dannyparsons** [comment]
no changes on this...

**Comment by maxwellfundi** [comment]
@FrancoisJRenaud  this should be closed. There no changes on this and it looks like an update of your repo from master

**Comment by dannyparsons** [comment]
Maybe you just haven't pushed changes to the branch.

**Comment by FrancoisJRenaud** [comment]
Ok sorry. I open the pull request as soon as I start working on it as people ay want to know which files I am editing. I then mentions when it is ready to be looked at. Is that bad practice ?

**Comment by dannyparsons** [comment]
It's a good idea actually, now we know about it anyway :)

File: instat/clsRLink.vb [hunk 1]
```diff
@@ -111,23 +111,19 @@ Public Class RLink
...
 
     Public Sub FillComboDataFrames(ByRef cboDataFrames As ComboBox, Optional bSetDefault As Boolean = True, Optional bIncludeOverall As Boolean = False, Optional strCurrentDataFrame As String = "")
-
-        If bInstatObjectExists Then
+        'This sub is filling the cboDataFrames with the relevant dat frame names (obtained by using GetDataFrameNames()) and potentially "[Overall]".  On thing it is doing, is setting the selected index in the cboDataFrames.
+        'It is used on the ucrDataFrame in the FillComboBox sub.
+        If bInstatObjectExists Then
             If bIncludeOverall Then
-                cboDataFrames.Items.Add("[Overall]")
+                cboDataFrames.Items.Add("[Overall]") 'Task/question: explain this.
```
Comment (dannyparsons) [inline instat/clsRLink.vb]: Remind me later to explain.

**Pull request merged**